src/main/scala/com/bwsw/cloudstack/vault/server/cloudstack/util/CloudStackEventHandler.scala

```diff
@@ -51,10 +51,10 @@ class CloudStackEventHandler(controller: CloudStackVaultController)
       } match {
         case Success(x) => x
         case Failure(e: JsonParseException) =>
-          logger.warn("Can not to parse record: \"" + s"$record" + "\", the empty CloudStackEvent is returned")
+          logger.warn("Can not to parse the record: \"" + s"$record" + "\", the empty CloudStackEvent will be returned")
```

Comment:
there is still a mistake, see the previous review

Reply:
"to" was removed